packages/discord.js/src/structures/MentionableSelectMenuInteraction.js

```diff
@@ -12,6 +12,12 @@ class MentionableSelectMenuInteraction extends MessageComponentInteraction {
   constructor(client, data) {
     super(client, data);
 
+    /**
+     * Array of mentioned ids
```

Comment:
Don't like how this doc comment is worded, it doesn't really say much either, this is a "mentionable select menu" since it lets you select roles and users.